src/components/shared/chart/Viewport.css

```diff
@@ -43,9 +43,9 @@
   text-overflow: ellipsis;
   will-change: opacity, transform;
   bottom: 0;
-  right: 0;
-  padding: 3px 10px;
-  border-radius: 5px 0 0 0;
+  left: 0;
+  padding: 3px 0px 3px 3px;
```

Comment:
After all the padding was right, I think... Sorry about that !

Reply:
@julienw So just to double check...

We want padding to be:
✅ padding: 3px 10px;
❌ padding: 3px 0px 3px 3px;

or leave it as is?